packages/components/src/Account/Auth/EmailAuth.tsx

```diff
@@ -13,12 +13,22 @@ import {
   ConfirmButtonContainer,
   AuthErrorMessage,
 } from "./AuthStyledComponents";
-import { isValidEmail } from "@joincivil/utils";
+import { isValidEmail, addToMailingList } from "@joincivil/utils";
```

Comment:
Seems like `addToMailingList` is not needed here?

Reply:
Argh, that's what I get for using auto add to import on VSCode. It automatically does this. I'll kill.